src/Data/Unrestricted/Internal/Consumable.hs

```diff
@@ -0,0 +1,33 @@
+{-# OPTIONS_HADDOCK hide #-}
+{-# LANGUAGE LinearTypes #-}
+
+module Data.Unrestricted.Internal.Consumable
+  (
+  -- * Consumable
+    Consumable(..)
+  , void
+  , lseq
+  , seqUnit
+  )
```

Comment:
Since these are internal modules, maybe it's better to export everything (and not have an export list at all)

Reply:
I put it there because it helps give a glance at what's defined in the module. I think it improves readability (at least for me, which might just be me) and can't hurt.